Tests/BusinessLayer/JsonMessageBuilder/JsonMessageBuilderTest.cpp

```diff
@@ -0,0 +1,1092 @@
+#include <gtest/gtest.h>
+#include <gmock/gmock.h>
+
+#include <QTest>
+#include <QDebug>
+#include <QJsonDocument>
+#include <QJsonObject>
+#include <QJsonArray>
+#include <QScopedPointer>
+
+#include "BusinessLayer/JsonMessageBuilder/JsonMessageBuilder.h"
+#include "BusinessLayer/JsonForwarder/JsonDefines.h"
+
+#include "Tests/DataLayer/KeyMotorData/MockKeyMotorData.h"
+#include "Tests/DataLayer/MotorDetailsData/MockMotorDetailsData.h"
+#include "Tests/DataLayer/MotorDetailsData/MockMotorDetailsUnit.h"
+#include "Tests/DataLayer/DriverControlsData/MockDriverControlsData.h"
+#include "Tests/DataLayer/MotorFaultsData/MockMotorFaultsData.h"
+#include "Tests/DataLayer/BatteryFaultsData/MockBatteryFaultsData.h"
+#include "Tests/DataLayer/BatteryData/MockBatteryData.h"
+#include "Tests/DataLayer/MpptData/MockMpptData.h"
+#include "Tests/DataLayer/MpptData/MockMpptUnit.h"
+#include "Tests/DataLayer/LightsData/MockLightsData.h"
+
+using ::testing::Return;
+using ::testing::ReturnRef;
+using ::testing::NiceMock;
+
+class JsonMessageBuilderTest : public ::testing::Test
+{
+};
+
+TEST(JsonMessageBuilderTest, keyMotor)
+{
+    JsonMessageBuilder jsonMessageBuilder;
+
+    // *INDENT-OFF*
```

Comment:
Does having indents on do something to the message?

Reply:
Yea, astyle makes it really unreadable.